book/http_cache.rst

```diff
@@ -1207,6 +1207,11 @@ One great advantage of the ESI renderer is that you can make your application
 as dynamic as needed and at the same time, hit the application as little as
 possible.
 
+.. caution::
+
+    The fragment listener only responds to signed requests. Requests are only
+    signed when using fragment renderer and the ``render_esi`` Twig function.
```

Comment:
Yeah, but as this section is about ESI specifically, I only mentioned `render_esi` to not make things more complex